rootfs/tpls/etc/rtorrent/.rtlocal.rc

```diff
@@ -47,4 +47,4 @@ log.add_output = "rpc_events","log"
 #log.xmlrpc = (cat,(cfg.logs),"xmlrpc.log")
 
 # Initialize plugins
-execute2 = {sh,-c,/usr/bin/php7 /var/www/rutorrent/php/initplugins.php rtorrent &}
+execute2 = {sh,-c,/usr/bin/php81 /var/www/rutorrent/php/initplugins.php rtorrent &}
```

Comment:
Can you open another PR for this change please?

Reply:
Done

Reply:
@blueberryapple I created a separate PR as requested by crazy-max, which has been merged already. See #204